pkg/proc/proc_test.go

```diff
@@ -1525,17 +1522,14 @@ func TestCondBreakpointError(t *testing.T) {
 		}
 
 		err = grp.Continue()
-		if err != nil {
-			if _, exited := err.(proc.ErrProcessExited); !exited {
-				t.Fatalf("Unexpected error on second Continue(): %v", err)
-			}
-		} else {
-			nvar := evalVariable(p, t, "n")
+		if err != nil && !errors.As(err, &proc.ErrProcessExited{}) {
```

Comment:
This is doing a different thing when err is ErrProcessExited (before it did nothing, now it's evaluating variables).

Reply:
Fixed.